cli/src/splinter/action/circuit/api.rs

```diff
@@ -33,7 +33,7 @@ impl<'a> SplinterRestClient<'a> {
     pub fn fetch_node_id(&self) -> Result<String, CliError> {
         Client::new()
             .get(&format!("{}/status", self.url))
-            .send()
+            .submit()
```

Comment:
Can you explain this change? Othere places in this file now also use send.

Reply:
Oops, think this was a mistake.